serialfile.go

```diff
@@ -109,7 +118,8 @@ func (f *serialFile) Entries() DirIterator {
 	return &serialIterator{
 		path:              f.path,
 		files:             f.files,
-		handleHiddenFiles: f.handleHiddenFiles,
+		filter:            f.filter,
+		handleHiddenFiles: f.filter.IncludeHidden,
```

Comment:
Do we still need the `handleHiddenFiles` field here? (we should just be able to call `it.filter.IncludeHidden` right?)